### DISCUSSION TITLE
EnvVarNotFound("Expected parent binding \'7\' to be present.")

### DISCUSSION THREAD

**Discussion by oieeaaaa** [prisma/prisma#4271] [Q&A] [answered]
Can anyone please tell me what does this error means?

This occurs when I use `connectOrCreate`. However, with `connect` or `create` only this works fine. 

Sample Code:
```ts
prisma.model.create({
     "unitOfMeasurement": {
        "connectOrCreate": {
            "where": {
                "name": "LUCID"
            },
            "create": {
                "name": "LUCID"
            }
        }
    },
})
```

**Comment by ryands17** [comment, marked as answer]
Hey @oieeaaaa 👋 

Answered [here](https://stackoverflow.com/questions/64867191/error-in-prisma-when-using-connectorcreate/)

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
